### ISSUE TITLE
Investigate behavior of the H2O credits tab to properly attribute works

### ISSUE THREAD

**Issue by cath9** [harvard-lil/h2o#1942]
When a remix is twice removed from the original (ie there is an original, a remix is made, and then another remix is made), the credits tab seems to drop credit to the original version/author, and give all credit to the first remix. We should maintain credit to the original author. 

First remix, with original attribution: https://opencasebook.org/casebooks/981-balloberman-crim-law-casebook/credits/
Second remix, without original attribution: https://opencasebook.org/casebooks/3553-balloberman-crim-law-casebook/credits/

**Comment by lizadaly** [comment]
I think this is working as designed (which may not be what's wanted now). Unfortunately the credits code has no comments or documentation so it's a little hard to follow.

Wu & Gersen are listed as "with additional contributions by" and in the code are referred to as "incidental authors". This is as opposed to "immediate authors", who are contributors to the immediate parent.

I think it makes sense to credit them differently because the code can't know how many intermediate changes have happened, but we could probably make it more prominent and/or link to the original casebook or section?

**Comment by lizadaly** [comment]
Will work on some improvements based on discussion and initial investigation.
